Packs/FiltersAndTransformers/Scripts/RemoveMatches/README.md

```diff
@@ -0,0 +1,48 @@
+Removes items from the given list of values if they match any of the patterns in the provided `filters`.
+  If the match_exact argument is 'yes', direct string compare is used, otherwise the comparison is done using regex.
+
+### Example:
+
+---
+
+##### value (Get):
+
+```text
+${.}
+```
+
+##### filters:
+
+```text
+^*.\.domain1.com/.*\n
+^*.\.domain2.com/.*\n
+^sub.domain3.com/.*
+```
```

Comment:
Please enter a list of values instead of ${.}, and an example of the excepted result in this case, so it will be clearer to the user what the transformer is doing.